README.md

```diff
@@ -1,7 +1,70 @@
 # AakSamlBundle
-Bundle to handle mapping between Aarhus Kommune SAML claims and Kimai team structure
+Bundle to handle mapping between City of Aarhus SAML claims and Kimai team structure.
 
-## Claims structure:
+Developed as a [Kimai](https://www.kimai.org/) plugin following the [developer documentation](https://www.kimai.org/documentation/plugins.html)
+
+## Mapping SAML claims to Kimai entities
+
+When a user logs in the following is created or updated. This is done by listening for a
+`Symfony\Component\Security\Http\Event\CheckPassportEvent` in `KimaiPlugin\AakSamlBundle\EventSubscriber\CheckPassportEventSubscriber` 
+which in turn delegates the heavy lifting to `KimaiPlugin\AakSamlBundle\Service\SamlDataHydrateService`.
+
+### `App\Entity\Team`
+- Office (`officeId`) -> `name`
+
+We map `Office` (e.g. "ITK Development") to a Kimai team. Kimai has a unique constraint on team names. We include the id
+to ensure uniqueness. E.g. "ITK Development (6530)"
+
+- The team MUST have one team lead only.
+- The team lead MUST be the user with email matching the `managerUPN` SAML claim.
+
+### `KimaiPlugin\AakSamlBundle\Entity\AakSamlTeamMeta`
+- Office -> `team`
+- SAML claims -> entity fields
+
+This plugin entity maps AAK organisation values to Kimai Teams. Most claims are mapped 1:1 through a `SamlDTO` object. 
+Note that `id's` for the various "departments" are given as a list in `extensionAttribute7`, e.g. `1001;1004;1012;1103;6530`. 
+These are split and handled as individual id's.
```

Comment:
```suggestion
These are split and handled as individual IDs.
```

(I think it's customary to write English abbreviations with uppercase letters, but I'm not sure).